test/Nancy.Tests/Unit/Sessions/CookieBasedSessionsFixture.cs

```diff
@@ -5,12 +5,12 @@ namespace Nancy.Tests.Unit.Sessions
     using System.Linq;
     using System.Reflection;
     using System.Threading;
-    using System.Web;
 
     using FakeItEasy;
 
     using Nancy.Bootstrapper;
     using Nancy.Cryptography;
+    using Nancy.Helpers;

```

Comment:
Nearly. It's an entire missing class. `System.Web.HttpUtility`. My understanding is that `System.Web` is not available in .NET Core.
`Nancy.Helpers.HttpUtility` is used in the production code, e.g. in `NancyCookie`, so I took advantage here. Are you concerned that it weakens the tests?